PlayerController/playercore.js

```diff
@@ -162,11 +162,11 @@ function markScrollPosition() {
 // Variable added by KV
 var gameName = document.title;
 
-function setGameName(text) {
+function setGameName(s) {

```

Comment:
Any reason for changing this variable name here?

Reply:
I was thinking `text` is a reserved keyword in some version of ECSMAScript, but I can't find anything about that when I look it up now. I guess it's `location` I was thinking of. Someone had one of these in a game that worked in the old 5.8 browser but not online.

I'll change this back while changing "deprecated" to "removed" on line 1319.